.github/workflows/main.yml

```diff
@@ -111,6 +111,39 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN }}
 
+  publish-rsconnect:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.8.x
+    - name: Install uv # see scripts/temporary-rename
+      uses: astral-sh/setup-uv@v5
```

Comment:
Set up Python using `uv`: https://github.com/posit-dev/posit-sdk-py/blob/main/.github/workflows/ci.yaml#L32-L33

Should be faster.

Reply:
Looks like the python version is builtin now as well: 

```yml
        uses: astral-sh/setup-uv@v6
        with:
          python-version: ${{ matrix.python-version }}

Reply:
I think that changes a few assumptions about how package installation works in the rest of the steps, which I'd prefer to leave in place for now. 

I think we could simplify much of the build infrastructure here out of docker and into `uv` and that's probably worth doing on its own.